mythril/analysis/modules/base.py

```diff
@@ -25,6 +25,16 @@ def __init__(
                 self.name,
             )
         self.entrypoint = entrypoint
+        self._issues = []
+        self.cache_addresses = {}
+
+    @property
+    def issues(self):
+        return self._issues
+
+    def reset_module(self):
+        self._issues = []
+        self.cache_addresses = {}
```

Comment:
`self._cache_addresses`? this is not something that needs to be publicly accessed right?

Reply:
Changed